openage/convert/interface/interfacecutter.py

```diff
@@ -24,34 +27,42 @@ def __call__(self, image):
         if is_ingame_hud_background(self.filename):
             pil_image = image.get_pil_image()
 
-            yield self.__cut_top_strip(pil_image)
+            yield self.__cut_strip(pil_image,
+                                   TOP_STRIP_PATTERN_CORNERS,
+                                   TOP_STRIP_PATTERN_SEARCH_AREA_CORNERS)
+
+            yield self.__cut_strip(pil_image,
+                                   MID_STRIP_PATTERN_CORNERS,
+                                   MID_STRIP_PATTERN_SEARCH_AREA_CORNERS)
 
             for coords in KNOWN_SUBTEX_CORNER_COORDS:
                 yield TextureImage(pil_image.crop(coords))
         else:
             yield image
 
-    def __cut_top_strip(self, pil_image):
+    def __cut_strip(self, pil_image, pattern_corners, search_area_corners):
```

Comment:
hm i think we can remove the `__`, and if you really wanna keep it, a single `_` should be enough.